package.json

```diff
@@ -33,39 +33,38 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^5.1.7",
+    "ember-cli-babel": "6.6.0",
     "ember-promise-tools": "1.0.0",
-    "ember-runtime-enumerable-includes-polyfill": "^1.0.1",
+    "ember-runtime-enumerable-includes-polyfill": "2.0.0",
     "ember-string-ishtmlsafe-polyfill": "^1.1.0"
   },
   "devDependencies": {
+    "babel-plugin-transform-es2015-block-scoping": "^6.24.1",
     "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "^2.4.1",
-    "ember-cli": "2.10.0",
-    "ember-cli-app-version": "^2.0.0",
-    "ember-cli-clipboard": "0.4.1",
-    "ember-cli-dependency-checker": "^1.3.0",
+    "ember-ajax": "3.0.0",
+    "ember-cli": "2.14.1",
+    "ember-cli-clipboard": "0.8.0",
+    "ember-cli-dependency-checker": "2.0.1",
+    "ember-cli-eslint": "^4.2.0",
     "ember-cli-github-pages": "0.1.2",
-    "ember-cli-htmlbars": "^1.0.10",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.3",
+    "ember-cli-htmlbars": "2.0.2",
+    "ember-cli-htmlbars-inline-precompile": "0.4.3",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-jshint": "^2.0.1",
     "ember-cli-qunit": "^3.0.2",
-    "ember-cli-release": "^0.2.9",
+    "ember-cli-shims": "^1.0.2",
     "ember-cli-sri": "^2.1.0",
-    "ember-cli-test-loader": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
-    "ember-composable-helpers": "1.1.2",
-    "ember-data": "^2.10.0",
+    "ember-composable-helpers": "2.0.3",
```

Comment:
Yes, still is used only here, but it can be removed in the future.

https://github.com/Semantic-Org/Semantic-UI-Ember/blame/master/tests/dummy/app/templates/components/ui-annotation.hbs#L2